releasenotes/notes/0.19/add-sparsepauliop-sum-d55fc817c9fded82.yaml

```
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Added :meth:`.SparsePauliOp.sum` as a specialized version of the builtin
+    function ``sum`` for a list of :class:`~qiskit.quantum_info.SparsePauliOp`.
+    This method has better scaling than adding the instances together in a loop.
```

It's a bit weird reading as everything is called `sum` maybe add an example of the builtin.

```suggestion
    Added the :meth:`.SparsePauliOp.sum` method as a specialized version of the builtin
    function ``sum`` (e.g. `sum([x, y])` ) for a list of :class:`~qiskit.quantum_info.SparsePauliOp`.
    This method has better scaling than adding the instances together in a loop.
```

I think this would also benefit an a/b example of what would have been done before and how the new `sum()` method can be used instead with better scaling.

---

Sure - in this case it'll display as
> Added `SparsePauliOp.sum` as a specialised version of the builtin function ``sum``, ...

(so the context is there), but I can make it clearer, and the example is good.